bulla-contracts/src/mappings/BullaFactoring.ts

```diff
@@ -284,11 +286,42 @@ export function handleDepositV2(event: Deposit): void {
   handleDeposit(event, "v2");
 }
 
-export function handleDepositMadeWithAttachment(event: DepositMadeWithAttachment): void {
+// @notice V1 does not have ERC4626 functions and events, so cannot reuse logic from V2
+export function handleDepositMadeWithAttachmentV1(event: DepositMadeWithAttachment): void {
   const ev = event.params;
+  const DepositMadeWithAttachmentEvent = createDepositMadeWithAttachmentEvent(event);
```

Comment:
to conform with the V2 design, we would create a normal DepositMade event and assign the ipfsHash to it.

Reply:
`Argument of type 'DepositMadeWithAttachment' is not assignable to parameter of type 'Deposit'.`

Reply:
that's why I added back `DepositMadeWithAttachmentEvent` in the schema